### ISSUE TITLE
Manually setting the learning_rate?

### ISSUE THREAD

**Issue by asmith26** [deepmind/optax#4]
Hi there,

Is it possible to set the learning rate manually? e.g.

```python
# Setup optimiser
opt_init, opt_update = optax.adam(learning_rate=1e-3)
opt_state = opt_init(params)

# Train
for epoch_num in range(10):
    # Compute gradients.
    grads = jax.grad(loss_function)(params, data)
    # Transform the gradients using the optimiser.
    updates, opt_state = opt_update(grads, opt_state)
    # Update parameters.
    params = optax.apply_updates(params, updates)

    # *** MY IDEA/INTENTION ***:
    if epoch_num == 5:
        opt_state.learning_rate = 1e-4  # does something like this exist?
```

Many thanks for any help, and for this fantastic lib! :)

**Comment by KristianHolsheimer** [comment]
You can either use a [schedule](https://github.com/deepmind/optax/blob/master/optax/_src/schedule.py) or create a new optimizer, e.g.
```python
_, opt_update = optax.adam(learning_rate=new_learning_rate)
``` 
The latter option is a bit blunt, but it works (as long as your `opt_state` is compatible with the new optimizer).

**Comment by mtthss** [comment]
You could:
* define a custom optimiser with `chain` and `scale_by_schedule`
* have two different optims with different learning rates (they are just functions so it doesn't cost you anything)

**Comment by asmith26** [comment]
Thanks @KristianHolsheimer and @mtthss for your help! 

I think I'll try the `_, opt_update = optax.adam(learning_rate=new_learning_rate)` approach. I thought I'd see if I can use reinforcement learning to automatically change the learning rate:
- actions = increase lr, decrease lr or no change,
- rewards = based on if the loss has improved or not.
- states (initial idea) = has the loss changed (e.g. after 5 epochs)
